themes/default/content/docs/concepts/inputs-outputs/apply.md

```diff
@@ -0,0 +1,818 @@
+---
+title_tag: "Accessing Single Outputs with Apply | Inputs and Outputs"
+meta_desc: "Learn how to access a single output value using the apply method in Pulumi."
+title: Accessing single outputs with Apply
+h1: Accessing single outputs with Apply
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  concepts:
+    weight: 2
+    parent: inputs-outputs
+---
+
+## Overview
+
+[Outputs](/docs/concepts/inputs-outputs/#outputs) are asynchronous, meaning their actual plain values are not immediately available. As such, there are limitations on the ways in which you can retrieve these values.
```

Comment:
I think it's worth making very clear that not only are outputs async but in some cases _there is no plain value_. E.g. during a preview the outputs for a new resource don't have any possible value.

Reply:
@Frassle Right now there is this note is at the bottom of the page:

{{< notes type="info" >}}
During some program executions, {{< pulumi-apply >}} doesn’t run. For example, it won’t run during a preview, when resource output values may be unknown. Therefore, you should avoid side-effects within the callbacks. For this reason, you should not allocate new resources inside of your callbacks either, as it could lead to `pulumi preview` being wrong.
{{< /notes >}}

Is this referring to the same thing? If so, should it be moved higher up?

Reply:
Yup same thing, I think that should be more prominent. A lot of users think output is harder than it ought to be because they think it's just an async.